### DISCUSSION TITLE
Custom mode when Game is open

### DISCUSSION THREAD

**Discussion by 4NXIE7Y** [BartoszCichecki/LenovoLegionToolkit#1419] [Q&A]
Is there any way to exclude an app LTT considers a game? I want the app Crosshair X not to enable custom mode.
![image](https://github.com/user-attachments/assets/cec1b852-c9ad-4491-9b14-f458b767460e)

**Comment by BartoszCichecki** [comment]
Game detection is based on what Windows reports. Try opening Xbox Game Bar when Crosshair X is open and uncheck "This is a game" or something similar.

**Reply by 4NXIE7Y** [reply]
Even Zen Browser is activating custom mode...
![image](https://github.com/user-attachments/assets/d2b8a747-4241-4770-8d6b-db8ed2554eca)
